### PR TITLE
fix a bunch of typos

### PR DESCRIPTION
**Pull request by ymo-sci**
fix some occurred typos

### CODE REVIEW

**Review by sabieber — APPROVED** [review]

**Review by fgoerner — APPROVED** [review]

**Review by meg1502 — APPROVED** [review]

File: src/main/java/sirius/biz/storage/layer2/jdbc/SQLDirectory.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
 /**
  * Stores the metadata of a {@link Directory} in the underlying JDBC database.
  * <p>
- * Note that all non trivial methods delegate to the associated {@link SQLBlobStorageSpace}.
+ * Note that all nontrivial methods delegate to the associated {@link SQLBlobStorageSpace}.
```
Comment (qw3ry) [inline src/main/java/sirius/biz/storage/layer2/jdbc/SQLDirectory.java:36]:
```suggestion
 * Note that all non-trivial methods delegate to the associated {@link SQLBlobStorageSpace}.
```

**Review by qw3ry — APPROVED** [review]

File: src/main/java/sirius/biz/storage/layer2/jdbc/SQLBlob.java [hunk 2]
```diff
@@ -171,6 +172,7 @@
      * Stores the last modification timestamp.
      */
     public static final Mapping LAST_MODIFIED = Mapping.named("lastModified");
+    @DefaultValue("")
     private LocalDateTime lastModified = LocalDateTime.now();
```
Comment (andyHa) [inline src/main/java/sirius/biz/storage/layer2/jdbc/SQLBlob.java:176]: shouldn't we always skip this for LocalDate & LocalDateTime properties? using the given default makes almost no sense anywhere (?)

Comment (meg1502) [reply src/main/java/sirius/biz/storage/layer2/jdbc/SQLBlob.java:176]: If we want to do this we can overwrite the column method in the properties classes, can copy from MultiLanguageStringProperty

Comment (ymo-sci) [reply src/main/java/sirius/biz/storage/layer2/jdbc/SQLBlob.java]: Is now fixed with https://github.com/scireum/sirius-db/pull/449 as suggested

**Review by meg1502 — APPROVED** [review]

**Pull request merged**